MIGRATION.md

```diff
@@ -182,6 +182,15 @@ For details and examples please refer to the official [Reservation Endpoint Docu
 | `required_order_data`         | added        |  N/A        |  `required_order_data`        |
 | `required_visitor_data`         | added        |  N/A        |  `required_visitor_data`     |
 
+#### Changes Affecting the Endpoint's Response
+
+| Field          | Type of Change | v1             | v2           |
+| -----------    | -----------    | -----------    | -----------  |
+| `unit_price`      | added          |  N/A           |  optional    |
```

Comment:
small column misalignment

Reply:
Markdown will still render the table properly.